src/wr_logic_ai/action/ShortRange.action

```diff
@@ -1,5 +1,5 @@
 uint8 target_type
-uint8 TARGET_TYPE_GPS_ONLY = 0
+uint8 TARGET_TYPE_GPS_ONLY = 0 # I remember being told to delete this, but I'm not sure.
```

Comment:
This relates to potential changes in how short range is structured. In previous years the autonomous challenge contains only GNSS-only waypoints and Aruco tag waypoints. This year they added additional waypoints with target objects that the rover would have to detect. We might make some changes to the state machine or short range action server because of this. For now, you can feel free to just leave it alone before Colin and I decide on a approach.